chezmoi: add page (#4314)
tldr-pages/tldr: pages/common/chezmoi.md

# Chezmoi

> A multi-machine dotfile manager, written in Go.
> More information: <https://github.com/twpayne/chezmoi>.

- Initialize chezmoi on your machine:

`chezmoi init`

- Tell chezmoi to manage a dotfile:

> `chezmoi add {{path_to_file}}`

Reviewer: we use `/`:
```suggestion
`chezmoi add {{path/to/file}}`
```

- Edit the source state of a tracked dotfile:

> `chezmoi edit {{path_to_file}}`

Reviewer: ```suggestion
`chezmoi edit {{path/to/file}}`
```

- See changes chezmoi would make:

`chezmoi diff`

- Apply the changes:

`chezmoi -v apply`

> - Set chezmoi up on another machine, from a service like GitHub:

Reviewer: I'm not sure about this description, the official site says:
```
Clone the git repo in ~/.local/share/chezmoi to a hosted Git service, e.g. GitHub, GitLab, or BitBucket. Many people call their dotfiles repo dotfiles. You can then setup chezmoi on a second machine:
```

Is this command for download dotfiles from your git repository?

Reviewer 2: It looks like it, @navarroaxel. What about this?
```suggestion
- Set chezmoi up on another machine by downloading existing dotfiles from a git repo:
```

> `chezmoi init {{https://git_url}}`

Reviewer: ```suggestion
`chezmoi init {{https://example.com/repository.git}}`
```

> - Fetch latest changes from a remote repo:

Reviewer 2: Perhaps this will read better?
```suggestion
- Fetch the latest changes from a remote repository:
```
